packages/flutter/test/gestures/drag_test.dart

```diff
@@ -1679,19 +1680,19 @@ testGesture('Touch drags should allow pointer pan/zooms to join them', (GestureT
     // Gesture will be claimed when distance reaches kPanSlop, which was 36.0 when this test was last updated.
     tester.route(panZoomPointer.panZoomUpdate(const Offset(10.0, 10.0), pan: const Offset(20.0, 20.0))); // moved 20 horizontally and 20 vertically which is 28 total
     expect(didStartPan, isFalse);
-    expect(updatedScrollDelta, const Offset(20.0, 20.0));
+    expect(updatedScrollDelta, isNull);
```

Comment:
Should panzoom be affected by this change? cc @moffatman

Reply:
Please see the conversation above.

Reply:
Yes, it should be affected. But I think this test might not make as much sense any more. Rather than changing the new numbers to match behaviour, you should be clear to understand what's being tested and perhaps change the sequence of events.

Reply:
> I tried it out in my app, works well 👍
> 
> FYI Pan/zoom = trackpad scrolling event. Zoom/scale is not relevant here, only pan, which could also be called scroll. When I implemented it, I summed the deltas as previous behavior, but your solution is better in my opinion. 

Thank you for your FYI.

>  The case of using both touchscreen and trackpad is not well handled by most platforms, trying it on iOS/UIKit causes a lot of flickering and is not correct, so there isn't any break with an existing standard to change it.

 So, the problem is the fixed only works well on the iOS and Android screens, but not on trackpad and touchscreen? I did not test on the trackpad, I will test it next week to improve. But I do not understand what's the problem with using a touchscreen, a lot of flickering on Andoird, iOS, or Windows surface? Can you give me some details about the devices to reproduce the problem? Thank you very much!

Reply:
@moffatman 
> Yes, it should be affected. But I think this test might not make as much sense any more. Rather than changing the new numbers to match behaviour, you should be clear to understand what's being tested and perhaps change the sequence of events.

I will update the tests this week.

Reply:
> Yes, it should be affected. But I think this test might not make as much sense any more. Rather than changing the new numbers to match behaviour, you should be clear to understand what's being tested and perhaps change the sequence of events.

@moffatman 

I have modified some tests, please have a review, thank you! If there is no problem, please say "Look good to me" to make me happy ![482A67FF](https://user-images.githubusercontent.com/35736246/177359316-2893198f-7f6c-4b6a-af5d-3b56af563119.png).